larq_compute_engine/mlir/python/common.cc

```diff
@@ -118,18 +115,10 @@ pybind11::bytes ConvertMLIRModuleToTFLiteFlatBuffer(
     }
   }
 
-  mlir::PassManager pm(&context, mlir::OpPassManager::Nesting::Implicit);
-  tensorflow::SetCrashReproducer(pm);
-
-  tensorflow::AddTFToLCETFLConversionPasses(quant_specs, &pm, target);
-
-  // Convert back to outlined while format for export back to flatbuffer.
-  pm.addPass(mlir::TFL::CreateWhileOutlinePass());
-  pm.addPass(mlir::TFL::CreateRuntimeVerifyPass());
-
```

Comment:
I see that this block has moved into `ConvertTFExecutorToFlatbuffer`, and `AddTFToLCETFLConversionPasses` is now split in two.
However `tensorflow::SetCrashReproducer(pm);`, `CreateWhileOutlinePass` and `CreateRuntimeVerifyPass` were not moved there, is that intentional? I see that `CreateRuntimeVerifyPass` is one of the passes inside `TFToLCETFL` so that's probably fine (although it's no longer the final pass).

Reply:
> However `CreateWhileOutlinePass` and `CreateRuntimeVerifyPass` were not moved there, is that intentional

These were moved to `tf_tfl_passes.cc` following https://github.com/tensorflow/tensorflow/commit/6347e46c2159293e7307a6ef71f7c8bc5ceec32a

> However `tensorflow::SetCrashReproducer(pm);` were not moved there, is that intentional

This was done following https://github.com/tensorflow/tensorflow/commit/a68046e7d2ee3396fee7cac07f915294619a15a7 so I assume that this functionality is now handled somewhere else internally.